packages/ckeditor5-html-embed/src/htmlembedediting.ts

```diff
@@ -167,7 +164,7 @@ export default class HtmlEmbedEditing extends Plugin {
 				} );
 
 				// API exposed on each raw HTML embed widget so other features can control a particular widget.
-				const rawHtmlApi = {
+				const rawHtmlApi: ContainerRawHtmlApiProperty = {
```

Comment:
Maybe this would be a better name?

```suggestion
				const rawHtmlApi: RawHtmlApi = {
```